src/components/shared/forms/fileUpload.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+
+import './styles/fileUpload.css';
```

Comment:
Please using scss directly.
```suggestion
import './styles/fileUpload.scss';
```